spec/controllers/groups_controller_spec.rb

```diff
@@ -241,6 +241,53 @@
       end
     end
 
+    describe '#create_groups_when_students_work_alone' do
+      before :each do
+        @assignment = create(:assignment)
+        # Create students
+        @student_user_names = %w[c8shosta c5bennet]
+        @student_user_names.each do |name|
+          create(:student, user: create(:end_user, user_name: name))
+        end
+      end
+
+      it 'creates groups for individual students' do
```

Comment:
The tests from here down to line 278 should be wrapped in a `context` that describes the situation (the assignment has a group max of 1).